tests/test_subscriptions.py

```diff
@@ -243,16 +222,17 @@ def throw_403(self):
 
 
 class TestGCPSubscription(TestSubscriptionsBase, unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        super().subsciption_setup(dss.Replica.gcp)
+    replica = dss.Replica.gcp
 
 
 class TestAWSSubscription(TestSubscriptionsBase, unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        super().subsciption_setup(dss.Replica.aws)
+    replica = dss.Replica.aws
 
+# Prevent unittest's discovery from attempting to discover the base test class. The alterative, not inheriting
+# TestCase in the base class, is too inconvenient because it interferes with auto-complete and generates PEP-8
+# warnings about the camel case methods.
+#
+del TestSubscriptionsBase
```

Comment:
Alternatively you could create subscriptionBase.py and import it into test_subscriptions.py

Reply:
See above.